src/components/LogIn.jsx

```diff
@@ -67,7 +78,7 @@ function LogIn() {
                         value={loginValues.password}
                         onChange={handleChange}
                     />
-                    <Button variant="contained">Let's get cook'n</Button>
+                    <Button onClick={onSubmit} variant="contained">Let's get cook'n</Button>
```

Comment:
if we call this function in the form tag like this `<form onSubmit={onSubmit}/>` I'm pretty sure this will allow users to also click enter after entering their credentials. Not 100% sure if it makes much of a difference having an onClick here though.